verrazzano/verrazzano
# VZ-10371 add the Thanos Ruler ingress

**Changes**
- Add the Thanos Ruler ingress if it is enabled
- Add Helm overrides to the Thanos component to support the new ingress
- Wire up the ingress with the authproxy
- Add the Thanos Ruler ingress to the Verrazzano Status field

**Testing**
- Install on a local kind cluster and make sure all of the links and the ingress works properly with authentication
- Install without enabling rule to make sure the ingress does not get created
